app/components/Breadcrumbs.tsx

```diff
@@ -54,27 +52,28 @@ export const Breadcrumbs = () => {
     </nav>
   );
 };
-const Breadcrumb = ({
-  title,
-  url,
-  icon,
-  disabled,
-}: Omit<BreadcrumbItem, "id">) => {
+const Breadcrumb = ({ title, url, disabled }: Omit<BreadcrumbItem, "id">) => {
```

Comment:
Nit: Let's add an empty line between the component definitions...
```suggestion

const Breadcrumb = ({ title, url, disabled }: Omit<BreadcrumbItem, "id">) => {
```